src/main/java/io/cdap/e2e/pages/locators/CdfConnectionLocators.java

```diff
@@ -94,4 +95,23 @@ public static WebElement locateConnectionAction(String connectionType, String co
 
   @FindBy(how = How.XPATH, using = "//div[@data-cy='connection-test-failure']")
   public static WebElement locateConnectionFooterError;
+
+
+  @FindBy(how = How.XPATH, using = "//img[@id='resource-center-btn']")
+  public static WebElement plusGreenButton;
+
+  public static By importuploadButton() {
+    return By.xpath("//input[@id='resource-center-import-pipeline']");
+  }
+
+  @FindBy(how = How.XPATH, using = "//*[@id='import-pipeline']")
+  public static WebElement importButton;
+
+  public static By locatorOfFixAll() {
```

Comment:
Instead of creating a string of ```public static String fixAllButtonXpath = "//button[@data-cy='fix-all-btn']"``` you can directly create a locator using xpath.

Reply:
resolved